99-fasschema.ldif

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
It looks like this file was wrongly added with the last commit.

Reply:
:man_facepalming: